specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2020-01-01-preview/examples/Operations_example.json

```diff
@@ -0,0 +1,113 @@
+              "operation": "Delete Cluster",
+              "description": "Delete any Cluster"
+            }
+          },
+          {
+            "name": "Microsoft.ServiceFabric/locations/clusterVersions/read",
+            "display": {
+              "provider": "Microsoft ServiceFabric",
+              "resource": "Cluster Versions",
+              "operation": "Read Cluster Version",
+              "description": "Read any Cluster Version"
+            }
+          },
+          {
+            "name": "Microsoft.ServiceFabric/managedclusters/locations/osType/clusterVersions/read",
+            "display": {
+              "provider": "Microsoft ServiceFabric",
+              "resource": "Cluster Versions By Os",
+              "operation": "Read Cluster Version By Os",
+              "description": "Read any Cluster Version for a specific os"
+            }
+          },
+          {
+            "name": "Microsoft.ServiceFabric/nodeTypes/read",
+            "display": {
+              "provider": "Microsoft ServiceFabric",
+              "resource": "nodeTypes",
+              "operation": "Read Node type",
+              "description": "Read any Node type"
+            }
+          },
+          {
+            "name": "Microsoft.ServiceFabric/nodeTypes/write",
+            "display": {
+              "provider": "Microsoft ServiceFabric",
+              "resource": "nodeTypes",
+              "operation": "Create or Update Node type",
+              "description": "Create or Update any Node type"
+            }
+          },
+          {
+            "name": "Microsoft.ServiceFabric/nodeTypes/delete",
+            "display": {
+              "provider": "Microsoft ServiceFabric",
+              "resource": "nodeTypes",
+              "operation": "Delete Node type",
+              "description": "Delete any Node type"
+            }
+          },
+          {
+            "name": "Microsoft.ServiceFabric/nodeTypes/restart/action",
+            "display": {
+              "provider": "Microsoft ServiceFabric",
+              "resource": "nodeTypes",
+              "operation": "Restart nodes from Node type",
+              "description": "Restart nodes from Node type"
+            }
+          },
+          {
+            "name": "Microsoft.ServiceFabric/nodeTypes/reimage/action",
+            "display": {
+              "provider": "Microsoft ServiceFabric",
+              "resource": "nodeTypes",
+              "operation": "Reimage nodes from Node type",
+              "description": "Reimage nodes from Node type"
+            }
+          },
+          {
+            "name": "Microsoft.ServiceFabric/nodeTypes/deleteNode/action",
+            "display": {
+              "provider": "Microsoft ServiceFabric",
+              "resource": "nodeTypes",
+              "operation": "Delete nodes from Node type",
+              "description": "Delete nodes from Node type"
+            }
+          }
+        ]
```

Comment:
I assume apllications and cluster types would also be part of this API version and those should also show up in the operations.

Reply:
we are not adding support for applications yet. This will probably come at a later api version.